lib/license_finder/package_manager.rb

```diff
@@ -111,11 +112,9 @@ def log_errors(stderr)
     end
 
     def log_to_file(contents)
-      log_dir = File.join(@project_path, 'lf_logs')
-      log_file_path = File.join(log_dir, 'error.log')
-
-      FileUtils.mkdir_p(log_dir)
-      File.open(log_file_path, 'w') do |f|
+      FileUtils.mkdir_p @log_directory
+      log_file = File.join(@log_directory, "#{self.class.package_management_command || 'errors'}.log")
```

Comment:
Just a minor thing and It may be very unlikely but some package managers might create log files based on their names

Reply:
We can prepend log files with 'prepare_' to avoid confusion